### DISCUSSION TITLE
Possible to get this on Cloud Servers?

### DISCUSSION THREAD

**Discussion by skzLA** [nefarius/ViGEmBus#176] [Ideas]
Trying to install this on AWS EC2 Nvidia T4 Windows Server 2019 and I get this error.

Anybody know of any work arounds?

![signal-2023-03-30-090944_002](https://user-images.githubusercontent.com/129416689/228931248-09a2ff01-8649-46ca-9ac4-7be62eff6cf1.png)

**Comment by nefarius** [comment]
Use the search next time, see #153

**Reply by skzLA** [reply]
Ah, my bad -- appreciate you responding so quickly. Just wanted to use my fight stick on cloud servers because the game I play has dog shit netcode, heh.
